src/Bicep.Core/Registry/AzureContainerRegistryManager.cs

```diff
@@ -172,79 +373,79 @@ private static void ValidateManifestResponse(Response<GetManifestResult> manifes
             }
         }
 
-        private static async Task<Stream> ProcessManifest(ContainerRegistryContentClient client, OciManifest manifest)
+        private static async Task<Stream> GetArmTemplateFromModuleManifestAsync(IOciRegistryContentClient client, OciManifest manifest)
```

Comment:
Can we move this function to a separate handler class?

Reply:
I assume you just mean this would be part of the Bicep-specific class from the refactoring I mentioned above?